version/version.go

```diff
@@ -2,11 +2,20 @@
 // describing the preflight project.
 package version
 
-import "fmt"
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/go-logr/logr"
+	"github.com/google/go-github/v57/github"
+	semvc "github.com/hashicorp/go-version"
```

Comment:
Is there any particular version why we're using this, and not https://pkg.go.dev/golang.org/x/mod/semver, or https://github.com/Masterminds/semver

Reply:
Is one of these already used in a project we manage? If so I'd vote for picking that one.

Reply:
changed to Masterminds 👍